src/renderer/features/network/index.ts

```diff
@@ -1,3 +1,5 @@
+export { addCustomRpcModel, AddCustomRpcModal, editCustomRpcModel, EditCustomRpcModal } from './ManageCustomRpcNode';
+export { RpcCheckResult } from './ManageCustomRpcNode/lib/types';
```

Comment:
Don't see any usage outside of Rpc feature 🤔

Reply:
Not with the utils any more indeed.